Assets/Scripts/AI/RacerManager.cs

```diff
@@ -13,7 +13,7 @@ public class RacerManager : MonoBehaviour
     private bool firstHit = true;
 
     private static float startTime = 0;
-    private static float timeElapsed = 0;
+    private static float timeElapsed;
```

Comment:
Given that you are not using timeElapsed += Time.time you don't need the startTime variable anymore